<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Playlist</class>
 <widget class="QWidget" name="Playlist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>310</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>180</width>
    <height>16777215</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="3">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="delButton">
     <property name="text">
      <string>Del</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="clearButton">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Playlist (Shortcut: L)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
